From 4448925930655dec57847ed41a34a24a8169d053 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Tue, 14 Jan 2014 17:52:01 +0100
Subject: [PATCH 28/29] BUILD/MINOR: listener: remove a glibc warning on
 accept4()

The accept4() Linux syscall requires _GNU_SOURCE on ix86, otherwise
it emits a warning. On other archs including x86_64, this problem
doesn't happen. Thanks to Charles Carter from Sigma Software for
reporting this.
---
 src/listener.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/listener.c b/src/listener.c
index afd6ae2..e5e723f 100644
--- a/src/listener.c
+++ b/src/listener.c
@@ -10,6 +10,7 @@
  *
  */
 
+#define _GNU_SOURCE
 #include <errno.h>
 #include <stdio.h>
 #include <string.h>
-- 
1.8.1.5

